packages/back-end/src/controllers/features.ts

```diff
@@ -535,9 +605,17 @@ export async function postFeaturePublish(
   req.checkPermissions("manageFeatures", feature.project);
 
   const revision = await getRevision(org.id, feature.id, parseInt(version));
+  const user = org.members.find((member) => member.id === userId);
+  const isAdmin = user?.role === "admin";
   if (!revision) {
     throw new Error("Could not find feature revision");
   }
+  if (
+    (org.settings?.requireReviews && !isAdmin) ||
+    revision.status === "approved"
```

Comment:
This seems backwards?  If the revision is approved, shouldn't publishing be allowed?